src/commands/deploy.ts

```diff
@@ -126,13 +130,30 @@ export async function deploy(context: IActionContext, confirmDeployment: boolean
 
     if (!defaultWebAppToDeploy && currentWorkspace && (isPathEqual(currentWorkspace.uri.fsPath, fsPath) || isSubpath(currentWorkspace.uri.fsPath, fsPath))) {
         // tslint:disable-next-line:no-floating-promises
-        node.treeItem.promptToSaveDeployDefaults(node, currentWorkspace.uri.fsPath, fsPath, context.properties);
+        treeItem.promptToSaveDeployDefaults(node, currentWorkspace.uri.fsPath, fsPath, context.properties);
     }
-    cancelWebsiteValidation(node.treeItem);
+    cancelWebsiteValidation(treeItem);
     await node.runWithTemporaryDescription("Deploying...", async () => {
-        // tslint:disable-next-line:no-non-null-assertion
-        await appservice.deploy(node!.treeItem.client, <string>fsPath, constants.extensionPrefix, context.properties);
+        await appservice.deploy(treeItem.client, <string>fsPath, constants.extensionPrefix, context.properties);
     });
+
+    const deployComplete: string = `Deployment to "${treeItem.client.fullName}" completed.`;
```

Comment:
Oh yeah, also you're not displaying the url anymore.

Reply:
So I did that intentionally because we now have 3 buttons and it felt cluttered with the url. I also feel like you don't need to see the url since you have 'Browse Website'. Thoughts?

Reply:
I agree with you.  I think I brought that up before, but I thought @fiveisprime or Chris wanted to keep it for visibility? Don't quote me on that though...